src/app/components/home/home.component.spec.ts

```diff
@@ -2,7 +2,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { HomeComponent } from './home.component';
-import { ComponentsModule } from '../../components/components.module';
+import { SharedModule } from '../../modules/shared/shared.module';
```

Comment:
Since we are using absolute paths, wouldn't this have to change as below?.
```suggestion
import { SharedModule } from 'src/app/modules/shared/shared.module';
```